arangod/StorageEngine/StorageEngine.h

```diff
@@ -88,6 +88,16 @@ struct Options;
 
 }  // namespace transaction
 
+class StorageSnapshot {
+ public:
+  StorageSnapshot() = default;
+  StorageSnapshot(const StorageSnapshot&) = delete;
+  StorageSnapshot& operator=(const StorageSnapshot&) = delete;
```

Comment:
Following our convention, it should rather be:
```suggestion
  StorageSnapshot(StorageSnapshot const&) = delete;
  StorageSnapshot& operator=(StorageSnapshot const&) = delete;
```